package.json

```diff
@@ -61,7 +61,7 @@
     "release-it": "^14.4.1",
     "ts-jest": "^26.5.2",
     "typedoc": "^0.20.28",
-    "typescript": "^4.1.5"
+    "typescript": "^4.4.0"
```

Comment:
updated typescript to be able to use [`useUnknownInCatchVariables` ](https://www.typescriptlang.org/tsconfig#useUnknownInCatchVariables). not sure how it was passing tests before, as this flag was introduced in 4.4.